packages/vite-plugin-shopify/README.md

```diff
@@ -87,7 +87,7 @@ In your `<head>` element add this
 Then render the `vite-tag` snippet (in your `<head>` element too) to insert tags for loading assets from a given entrypoint file:
 
 ```liquid
-{% render 'vite-tag' with 'theme.ts' %}
+{% render 'vite-tag' with 'entrypoints/theme.ts' %}
```

Comment:
I thought it was kinda nice to only have to specify the filename when following the most basic use-case of generating tags only for files under the entrypoints dir. this is also technically a breaking change so we would need to update sites that are using this plugin already. (just B+D at the moment, but still)